### PR TITLE
dropped geographies from mds-db

### PR DESCRIPTION
**Pull request by nessur**
## 📚 Purpose
*[In order to completely  delete geographies from mds-db, i had to replace their usage in compliance-engine and compliance-batch-processor. Also, the utility method getPolygon needed to be updated to use GeographyDomainModel, and thus had to be moved to mds-geography-service in order to avoid a circular reference collapse in the JS build.]*

**ALSO REMOVED `Geography` from `@mds-types`**

## 👌 Resolves:
- [ ] ✨ PLAT-356 drop geographies from mds-db

## 📦 Impacts:
*[mds-compliance-engine]*
*[mds-compliance-batch-processor]*
*[mds-geography-service]*
*[mds-geography-author-api]*
*[mds-geography-api]*
*[mds-types]*
*[mds-schema-validators]*
*[mds-utils]*
*[mds-db]*

### CODE REVIEW

**Review by avatarneil — APPROVED** [review]
LGTM! Do you wanna also kill the `Geography` type in mds-types in this PR?

File: packages/mds-compliance-batch-processor/package.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
     "@mds-core/mds-compliance-engine": "workspace:0.7.12",
     "@mds-core/mds-compliance-service": "workspace:0.10.6",
     "@mds-core/mds-db": "workspace:0.4.32",
```
Comment (PeterSP) [inline packages/mds-compliance-batch-processor/package.json:25]: Do we have anything that'll alert us when we don't need mds-db anymore in a package?

Comment (avatarneil) [reply packages/mds-compliance-batch-processor/package.json:25]: Nope

File: packages/mds-compliance-engine/tests/engine/engine-test.spec.ts [hunk 2]
```diff
@@ -38,9 +39,9 @@ let policies: PolicyDomainModel[] = []
 
 const CITY_OF_LA = '1f943d59-ccc9-4d91-b6e2-0c5e771cbc49'
 
-const geographies: Geography[] = [
+const geographies = [
   { name: 'la', geography_id: CITY_OF_LA, geography_json: LA_CITY_BOUNDARY as FeatureCollection }
-]
+] as GeographyDomainModel[]
```
Comment (PeterSP) [inline packages/mds-compliance-engine/tests/engine/engine-test.spec.ts:44]: Why the change to `as` casting?

**Review by PeterSP — APPROVED** [review]

File: packages/mds-compliance-engine/tests/engine/engine-test.spec.ts (hunk 2 above)
Comment (nessur) [reply packages/mds-compliance-engine/tests/engine/engine-test.spec.ts:44]: for brevity, to quickly convert these fixtures to `GeographyDomainModel`, without appending the required-but-nullable fields.
